docs/source/python/api/compute.rst

```diff
@@ -105,8 +104,6 @@ variants which detect domain errors.
    log1p_checked
    log2
    log2_checked
-   logb
-   logb_checked
```

Comment:
These kernels definitely exist in the development branch. We should not remove them from the list.